$NetBSD: patch-lib_CMakeLists.txt,v 1.2 2021/01/28 19:46:24 triaxx Exp $

On NetBSD, do not include librt.
\todo Why?  All other "UNIX" does, except Mac.
\todo Consider filing upstream.

--- lib/CMakeLists.txt.orig	2021-01-11 16:49:40.000000000 +0000
+++ lib/CMakeLists.txt
@@ -61,12 +61,12 @@ set(C_SRC
 
 set (LIBRARIES ${OPENSSL_LIBRARIES} ${PTHREAD_LIBRARIES})
 
-if (UNIX AND NOT APPLE AND NOT ANDROID)
+if (UNIX AND NOT APPLE AND NOT ANDROID AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 	find_library(LIBRT rt)
 	if (LIBRT)
 		set (LIBRARIES ${LIBRARIES} rt)
 	endif (LIBRT)
-endif (UNIX AND NOT APPLE AND NOT ANDROID)
+endif (UNIX AND NOT APPLE AND NOT ANDROID AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 
 if (WIN32)
 	set (LIBRARIES ${LIBRARIES} ws2_32)
